scripts/update-image.py

```diff
@@ -801,14 +801,20 @@ def setup_esp(start, size):
 
             # manipulate it using mtools
             dosimg = "{}@@{}".format(self.image, start * 512)
-            # create \EFI
+            # create \EFI\BOOT
             logged_check_call(["mmd", "-i", dosimg, "EFI"])
+            logged_check_call(["mmd", "-i", dosimg, "EFI/BOOT"])
+            # HACK(qookie): We add CREATE_DIR tasks for these two...
+            # create \managarm
```

Comment:
hmm, why?

Reply:
We create `\managarm` and `\grub` because files get copied there (the kernel & initrd, and `grub.cfg` respectively), which fails if they don't exist.